examples/05_glm_second_level/plot_proportion_activated_voxels.py

```diff
@@ -26,8 +26,7 @@
 data = fetch_localizer_contrasts(
     ["left vs right button press"],
     n_subjects,
-    get_tmaps=True,
-    legacy_format=False,
+    legacy_format=False
```

Comment:
```suggestion
    legacy_format=False,
```

to fix black failure